### DISCUSSION TITLE
LTE Validation on TextInput

### DISCUSSION THREAD

**Discussion by falconsmilie** [filamentphp/filament#6818] [Help] [answered]
Hi.

This valiation code **_doesn't_** work for me;
```
                   TextInput::make('max')
                        ->numeric()
                        ->lte(fn () => 1000)

                   // or

                   TextInput::make('max')
                        ->numeric()
                        ->lte(1000)
```

I get this response: `The max must be less than or equal to data.1000.` even when the numbers are beneath 1000.

But this code **_does_** work;
```
                    TextInput::make('max')
                        ->rules(['numeric', 'lte:1000']),
```

I'd like to use the call to `lte()` on the field because being able to provide a callback means i can have a dynamic number to pass to `lte()`.

There are a number of other validators on this form's other fields, and they all function correctly. We are using `"filament/filament": "^2.0"`.

Can anyone advise what i am doing wrong with `lte` please?

**Comment by danharrin** [comment, marked as answer]
Wrong method, should prob be `maxValue(1000)`. `lte()` is for comparing the current field to a different field
